templates/explore/repo_search.tmpl

```diff
@@ -21,7 +42,7 @@
 		</div>
 	</div>
 </div>
-<form class="ui form ignore-dirty" style="max-width: 90%">
+<form class="ui form ignore-dirty" style="max-width: 70%">
```

Comment:
```suggestion
<form class="ui form ignore-dirty" >
```
`style="max-width: 70%"` please avoid the inline styles

Reply:
We removed these already in master branch.

Reply:
removed all the inline styles